Extensions/CMake/SlicerExtensionPackageAndUploadTarget.cmake

```diff
@@ -6,10 +6,11 @@
 # extension.
 #
 # The new target that will
-#  (1) build the standard 'package' target,
-#  (2) extract the list of generated packages from its standard output,
-#  (3) append the list of generated package filepaths to a file named PACKAGES.txt,
-#  (4) upload the first package.
+#  (1) configure the project and build the standard 'package' target,
+#  (2) extract metadata from the locally generated .s4ext file
```

Comment:
do we still have .s4ext file?

Reply:
Only internally for now, I would like to move forward in stages.